packages/core/src/parse-html/parse-elem-html.ts

```
@@ -38,12 +40,18 @@ function parseElemHtml($elem: Dom7Array, editor: IDomEditor): Descendant | Desce
+      return Array.from(childNodes).reduce((acc, child) => {
         const $childElem = $(child)
 
-        if (isDOMText($childElem[0])) { return { text: $childElem[0].textContent || '' } }
-        return parseTextElemHtml($childElem, editor)
-      })
+        if (isDOMComment($childElem[0])) { return acc }
+        if (isDOMElement($childElem[0])) {
+          const v = parseElemHtml($childElem, editor)
+
+          if (Array.isArray(v)) { acc = [...acc, ...v] } else { acc.push(v) }
+        }
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)